internal/keepers/ocr.go

```diff
@@ -11,6 +11,14 @@ import (
 	"golang.org/x/crypto/sha3"
 )
 
+const (
+	// observationKeysLimit is the max number of keys that Observation could return.
+	observationKeysLimit = 1
+
+	// reportKeysLimit is the maximum number of upkeep keys returned by the report phase
```

Comment:
nit. reportKeysLimit is the maximum number of upkeep keys checked during the report phase

Reply:
Done